apps/tailwind-components/app/composables/fetchTableData.ts

```diff
@@ -46,7 +46,7 @@ export default async (
     method: "POST",
     body: {
       query,
-      variables: { filter, orderby },
+      variables: { filter, orderby, limit },
```

Comment:
why would we need to pass the limit again via the variables when it's already bound to the query

Reply:
is it? I will check again (maybe by adding a test) but I think I observed that limits were not applied.

Reply:
I think you might be right. Should be there so will dig a bit deeper.